mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,27 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+  if (num > 10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+  let index = arr.slice(3, 4);
+  let element = index;
 
-  return element;
+  return element[0];
```

Comment:
I think it would be easier to use `return arr[x]`
where x is the correct index number.

Reply:
Thanks Karen. I understand this part now. I've change it.